tests/fixtures/requirements-locked.txt

```diff
@@ -34,6 +34,8 @@ tomli @ https://files.pythonhosted.org/packages/97/75/10a9ebee3fd790d20926a90a25
 -e /tmp/editable ; python_version >= "3.7" and python_version < "3.12"
 
 --index-url https://mirror1.phylum.io/simple/
--i https://mirror2.phylum.io/simple/
+other-registry-a==3.2.1
+-i=https://mirror2.phylum.io/simple/
```

Comment:
For what it's worth, this one isn't actually valid

```
> cat req.txt
-i=https://mirrors.sustech.edu.cn/pypi/simple
pytoml==0.1.21

> pip install --dry-run -r req.txt
WARNING: The index url "=https://mirrors.sustech.edu.cn/pypi/simple" seems invalid, please provide a scheme.
Looking in indexes: =https://mirrors.sustech.edu.cn/pypi/simple
WARNING: Location '=https://mirrors.sustech.edu.cn/pypi/simple/pytoml/' is ignored: it is either a non-existing path or lacks a specific scheme.
ERROR: Could not find a version that satisfies the requirement pytoml==0.1.21 (from versions: none)
ERROR: No matching distribution found for pytoml==0.1.21
```

Reply:
But annoyingly... that means this one **is valid** 🫠

```
> cat req.txt
-ihttps://pypi.org/simple
pytoml==0.1.21

> pip install --dry-run -r req.txt
Collecting pytoml==0.1.21
  Downloading pytoml-0.1.21-py2.py3-none-any.whl (8.5 kB)
Would install pytoml-0.1.21
```

Reply:
Like this:
<img width="1599" alt="image" src="https://github.com/phylum-dev/cli/assets/18729796/73832654-d84b-4a56-8dc1-20c206de46df">

Reply:
Sorry... My original comment was unclear. `-i=` is not valid, but `--index-url` is valid.

There are 4 accepted forms:

```
--index-url https://pypi.org/simple
--index-url=https://pypi.org/simple
-i https://pypi.org/simple
-ihttps://pypi.org/simple
```